tracer/test/Datadog.Trace.Tests/Telemetry/TelemetrySettingsTests.cs

```diff
@@ -80,13 +74,15 @@ public void WhenApiKeyAndDdSiteIsProvided_UsesDdSiteDomain()
                 { ConfigurationKeys.Site, domain },
             });
 
-            var settings = new TelemetrySettings(source, _tracerSettings);
+            var settings = TelemetrySettings.FromSource(source);
 
-            settings.TelemetryUri.Should().Be($"https://instrumentation-telemetry-intake.{domain}/");
+            settings.Agentless.Should().NotBeNull();
+            settings.Agentless.AgentlessUri.Should().Be($"https://instrumentation-telemetry-intake.{domain}/");
+            settings.ConfigurationError.Should().BeNullOrEmpty();
         }
 
         [Fact]
-        public void WhenInvalidUrlApiIsProvided_AndNoApiKey_UsesDefaultAgentUrl()
+        public void WhenInvalidUrlApiIsProvided_AndNoApiKey_AgentlessIsNotEnabled()
```

Comment:
```suggestion
        public void WhenInvalidUrlIsProvided_AndNoApiKey_AgentlessIsNotEnabled()
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2617/commits/268f3caa33852467feec3a28e3e528b0011dfa40